pkg/kubelet/cm/admission/errors.go

```diff
@@ -49,14 +58,56 @@ func GetPodAdmitResult(err error) lifecycle.PodAdmitResult {
 		return lifecycle.PodAdmitResult{Admit: true}
 	}
 
+	var errs []error
```

Comment:
I would not expect this to bubble all the way up to the generic admission handling. Doing so creates unnecessary coupling. I think this should be handled in `TopologyManager.Admit`. You could change the method signature of `Scope.Admit` to return an `error` rather than a `lifecycle.PodAdmitResult`.